Quora/models.py

```diff
@@ -41,7 +45,35 @@ class User(AbstractUser):
     username = None
     email = models.EmailField(('email address'), unique=True)
 
+    # Should add list of posts
+
     USERNAME_FIELD = 'email'
     REQUIRED_FIELDS = []
 
-    objects = UserManager()
\ No newline at end of file
+    objects = UserManager()
+
+
+class Post(models.Model):
+    """Post model."""
+    id = models.AutoField(primary_key=True)
+    post_date = models.DateTimeField(default=datetime.now)
+    user = models.ForeignKey(User, null=True, on_delete=models.CASCADE)
+    question = models.TextField()
+
+class Answer(models.Model):
+    """Answer model"""
+    id = models.AutoField(primary_key=True)
+    date = models.DateTimeField(default=datetime.now)
+    original_post = models.ForeignKey(
+        Post, null=False, on_delete=models.CASCADE)
+    user = models.ForeignKey(User, null=True, on_delete=models.CASCADE)
+    content = models.TextField()
+
+
+'''
+class PostAnswers(models.Model):
+    ref_post_id = #?
+    answer_content = models.CharField()
+    answer_date = models.DateTimeField()
+    answer_user = #? user who answers
+'''
```

Comment:
No need for this anymore as `Answer` is already implemented

Reply:
It's a comment, I forgot to delete it.

Reply:
No prob, I cleaned it up so we don't carry it over when this PR gets merged.